packages/backend/server/src/plugins/copilot/context/session.ts

```diff
@@ -63,15 +62,24 @@ export class ContextSession implements AsyncDisposable {
   }
 
   async addCategoryRecord(type: ContextCategories, id: string, docs: string[]) {
-    const existDocs = await this.models.doc.existsAll(this.workspaceId, docs);
-    if (!existDocs) {
-      throw new CopilotDocsNotFound();
-    }
-
     const category = this.config.categories.find(
       c => c.type === type && c.id === id
     );
     if (category) {
+      const missingDocs = docs.filter(
+        docId => !category.docs.some(d => d.id === docId)
+      );
```

Comment:
Better to use lodash difference like:
```typescript
_.difference(docs, _.map(category.docs, 'id'))
```

Reply:
By the way, missing docs are not equal to not existing docs?

Reply:
not equal, not existing docs already exclude at job running

Reply:
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/4d6a7b76-2f6c-4846-ab7b-6d014d575dac" />

poor performance